src/Commands/CreatePdf.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Drupal\os2forms_digital_post\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+use Drupal\os2forms_digital_post\Manager\TemplateManager;
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+/**
+ * A drush command file.
+ *
+ * @package Drupal\event_database_pull\Commands
```

Comment:
You may want to mention “os2forms_digital_post” here …

Reply:
Added